packages/dashboard-frontend/src/store/__mocks__/devWorkspaceBuilder.ts

```diff
@@ -10,33 +10,36 @@
  *   Red Hat, Inc. - initial API and implementation
  */
 
-import { IDevWorkspace } from '../../services/workspace-client/devworkspace/types';
+import devfileApi from '../../services/devfileApi';
 import getRandomString from '../../services/helpers/random';
 import { DevWorkspaceStatus, WorkspaceStatus } from '../../services/helpers/types';
 
 export class DevWorkspaceBuilder {
 
-  private workspace: IDevWorkspace = {
+  private workspace: devfileApi.DevWorkspace = {
     kind: 'DevWorkspace',
     apiVersion: 'workspace.devfile.io/v1alpha2',
     metadata: {
+      annotations: {},
+      labels: {},
       name: 'dev-wksp-' + getRandomString(4),
       namespace: '',
+      uid: 'uid-' + getRandomString(4)
     },
     spec: {
       started: false,
       routingClass: 'che',
       template: {},
     },
     status: {
-      devworkspaceId: getRandomString(4),
+      devworkspaceId: 'devworkspaceId-' + getRandomString(4),
       mainUrl: '',
       phase: 'STOPPED',
     }
   }
 
   withId(id: string): DevWorkspaceBuilder {
-    this.workspace.status.devworkspaceId = id;
+    this.workspace.metadata.uid = id;
```

Comment:
It is not the same 'status.devworkspaceId' and 'metadata.uid'.
We need to recover a new value for ```metadata.uid``` from the ```stratus.devworkspaceId```.

Reply:
well, for devworkspaces I would say we need to drop workspace id notion and use namespace/name instead.

Reply:
but yeah, to move in an iterative manner, it should be as Orel provided https://github.com/eclipse-che/che-dashboard/pull/329#discussion_r702012154
the original https://github.com/devfile/devworkspace-operator/blob/89eb71a53c5e61b643d93d8f07530d32df7613d9/controllers/workspace/devworkspace_controller.go